src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__wrapper calendar__wrapper--month-length-30">
+        <div class="calendar__day calendar__day--1 calendar__day--mon"></div>
+        <div class="calendar__day calendar__day--2"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
modifiers for calendar_day are redundant